CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.6.0
   - [CHANGE] `AllowOverlappingBlock` is now `AllowOverlappingBlocks`.
+  - [BUGFIX] Update `last` after appending a non-overlapping chunk in `chunks.MergeOverlappingChunks`. [#539](https://github.com/prometheus/tsdb/pull/539)
```

Comment:
Can you move this under `## master / unreleased`? And I will merge it. Thanks for your work!

Reply:
I did it myself, sorry for hijacking.